src/mavlink_server.h

```diff
@@ -54,6 +54,7 @@ class MavlinkServer {
     char *_rtsp_server_addr;
     RTSPServer &_rtsp;
     std::map<int, CameraComponent *> compIdToObj;
+    bool _is_conf_sysid;
 
```

Comment:
We can get rid of this flag. Either the sysid comes from conf file or heartbeat, else it stays invalid. Order of preference could be 1) heartbeat 2) conf file.